gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title:  "Ideas for IMS - ERAS"
+show_main: false
+ideas_team: IMS
+---
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+*Suggested Mentors:* [Antonio del Mastro](https://bitbucket.org/aldebran/) , [Olena Persianova](https://bitbucket.org/OPersian/)
```

Comment:
I have no knowledge whatsoever of astronomy, let alone solar phenomena.
I'm a computer scientist with an specialization in machine learning, but in this instance I'm just the guy adding this project proposal to OpenAStronomy.
That aside, it's very kind of you to offer help :) I'll pass along the message